metricbeat/module/windows/perfmon/perfmon.go

```diff
@@ -102,3 +100,12 @@ func (m *MetricSet) Fetch(report mb.ReporterV2) {
 		report.Event(event)
 	}
 }
+
+// Close will be called when metricbeat is stopped, should close the query.
+func (m *MetricSet) Close() error {
```

Comment:
Thanks for implementing this closer :+1: 

It could be also written in one line as:
```
return errors.Wrap(m.reader.Close(), "failed to close perfmon reader"))
```
But I don't have a strong preference, so as you prefer.